<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWindow</class>
 <widget class="QMainWindow" name="TableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>테이블 조회</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="titleFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>971</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="searchPushButton">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchTotalPushButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>20</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>전체조회</string>
     </property>
    </widget>
    <widget class="QLabel" name="portNumberText">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>66</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string> 포트 번호</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portNumberComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="arpFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>971</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="arpTableText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell Extra Bold</family>
       <pointsize>9</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(94, 92, 100);</string>
     </property>
     <property name="text">
      <string>ARP 테이블</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="arpTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>971</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(94, 92, 100);</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>242</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>28</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string> 포트 번호</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP Address</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC 주소</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>타입</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="arpText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cantarell Extra Bold</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(94, 92, 100);</string>
    </property>
    <property name="text">
     <string>ARP 테이블 조회</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
